### DISCUSSION TITLE
Wrongly align tasks

### DISCUSSION THREAD

**Discussion by Praczet** [GothenburgBitFactory/taskwarrior#3610] [Community support]
Hi,
I am quite new to the taskwarrior and I have say I like it a lot.

I notice that  tasks  when using an Unicode character outside the Basic Multilingual Plane are not perfectly aligned
For example character:󰌵 
screen:
![image](https://github.com/user-attachments/assets/9a1abd67-f7b3-43e2-b51e-a5f319d36ecb)

I believe this is due to the fact that those characters take up more bytes than a standard ASCII character.
I noticed this when I was displaying tasks in my nvim plugin. normal method to calculate string length #string didn't work or rather it gives number of byte used in string then number of characters

in C++ you calculate it like this.

```cpp
#include <iostream>
#include <unicode/unistr.h>

int main() {
    icu::UnicodeString b = UNICODE_STRING_SIMPLE("󰌵est");

    std::cout << "Length of b (code points): " << b.length() << std::endl;  // Output: 4

    return 0;
}
```

Best regards
Adam

**Comment by djmitche** [comment]
There was some related discussion in https://github.com/GothenburgBitFactory/taskwarrior/issues/2599. I wonder if this character is new since the latest update of the library updated in https://github.com/GothenburgBitFactory/taskwarrior/issues/2333?
